src/main/java/org/dependencytrack/resources/v1/vo/BomSubmitRequest.java

```diff
@@ -51,6 +53,8 @@ public final class BomSubmitRequest {
     @Pattern(regexp = RegexSequence.Definition.PRINTABLE_CHARS, message = "The project version may only contain printable characters")
     private final String projectVersion;
 
+    private final List<Tag> projectTags;
```

Comment:
We should probably accept a list of `String` here instead. From the client's perspective, passing this:

```json
{
  "projectTags": ["foo", "bar"]
}
```

is more straightforward than:

```json
{
  "projectTags": [
    {
      "name": "foo"
    },
    {
      "name": "bar"
    }
  ]
}
```

Reply:
I agree, but I also think that using Tag keeps the API consistent, as in other methods this happens.

For example:
```json
PUT /v1/project

{
  "tags": [
    {
      "name": "LNsN}MPtIehaapsLeihSsMtMctLN{_aNcipcseIaLSWaNs}t_IesSSSsicpe_hIWtL_McIhs{L}P_SPc}S_M}{S}peWhcP_PeWhs"
    }
  ],
  ...
}
```


Do I make the change to make it easier or do I leave it like this to make the api more consistent?

Reply:
Good point. Let's leave it consistent then.